services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -45,7 +45,7 @@ class PostgresSettings(BaseSettings):
     port: int = 5432
 
     minsize: int = 10
-    maxsize: int = 10
+    maxsize: int = 30
```

Comment:
could you add a comment expressing what do these refer to?

Reply:
Maximum number of connections in the [db engine pool](https://aiopg.readthedocs.io/en/stable/core.html#pool)

The pg accepts up to 100 connections that must be distributed among different services. See Process List in adminer:
![image](https://user-images.githubusercontent.com/32402063/95168972-65e4d200-07b2-11eb-99ff-845de43b3cd7.png)